app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  after_create :update_likes_counter
```

Comment:
optional:
you should make the `:update_likes_counter` method private and you should use the rails callback `after-save` to call the method as mentioned above every time a new post is saved to the database 👍🏾